adapter/config/ingress/status.go

```diff
@@ -68,14 +71,21 @@ func NewStatusSyncer(mesh *proxyconfig.ProxyMeshConfig, client kubernetes.Interf
 	if mesh.IngressService != "" {
 		publishService = fmt.Sprintf("%v/%v", options.Namespace, mesh.IngressService)
 	}
+	glog.Infof("ANDRA STATUS publishService ", publishService)
 	ingressClass, defaultIngressClass := convertIngressControllerMode(mesh.IngressControllerMode, mesh.IngressClass)
+
+	customIngressStatus := func(*betaext.Ingress) []v1.LoadBalancerIngress {
+		return nil
+	}
+
 	sync := status.NewStatusSyncer(status.Config{
 		Client:              client,
 		IngressLister:       store.IngressLister{Store: informer.GetStore()},
 		ElectionID:          ingressElectionID, // TODO: configurable?
 		PublishService:      publishService,
 		DefaultIngressClass: defaultIngressClass,
 		IngressClass:        ingressClass,
+		CustomIngressStatus: customIngressStatus,
```

Comment:
Yes, it is related to upgrading the ingress.
This could be done separately, but giving that we did not enforce this rule in the past, and we approved reviews containing refactoring + fixes for unrelated features, I will claim the same privilege and save the extra effort.